spec/requests/api/v1/facts_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'swagger_helper'
+
+RSpec.describe 'api/v1/facts', type: :request do
+  # Initialize the test data
```

Comment:
Some people don't like comments being left in code, however I find that in larger spec files, they can be helpful in breaking up some of the different parts of the tests to more quickly see what is happening in that section! You can even add some additional notes that briefly describe what is happening in each portion of this spec file.